subscriber.go

```diff
@@ -30,43 +28,42 @@ func subscriber(msg *nats.Msg) {
 
 	log.Printf("received: %s", msg.Subject)
 
-	// get graph and processed component
+	// TODO : This will return nil in case an error getting the graph occurs,
+	// is ok to not skip the process here?
 	scheduler.graph = m.getGraph()
```

Comment:
This will return nil in case an error getting the graph occurs, is ok to not skip the process here?